doc/classes/RigidBody2D.xml

```diff
@@ -114,64 +114,64 @@
 		</method>
 	</methods>
 	<members>
-		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp">
+		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp" default="-1.0">
 			Damps the body's [member angular_velocity]. If [code]-1[/code], the body will use the [b]Default Angular Damp[/b] defined in [b]Project &gt; Project Settings &gt; Physics &gt; 2d[/b]. Default value: [code]-1[/code].
 		</member>
-		<member name="angular_velocity" type="float" setter="set_angular_velocity" getter="get_angular_velocity">
+		<member name="angular_velocity" type="float" setter="set_angular_velocity" getter="get_angular_velocity" default="0.0">
 			The body's rotational velocity.
 		</member>
-		<member name="applied_force" type="Vector2" setter="set_applied_force" getter="get_applied_force">
+		<member name="applied_force" type="Vector2" setter="set_applied_force" getter="get_applied_force" default="Vector2( 0, 0 )">
 			The body's total applied force.
 		</member>
-		<member name="applied_torque" type="float" setter="set_applied_torque" getter="get_applied_torque">
+		<member name="applied_torque" type="float" setter="set_applied_torque" getter="get_applied_torque" default="0.0">
 			The body's total applied torque.
 		</member>
-		<member name="bounce" type="float" setter="set_bounce" getter="get_bounce">
+		<member name="bounce" type="float" setter="set_bounce" getter="get_bounce" default="null">
```

Comment:
Bug here. (Why does RigidBody2D still have a bounce property though, shouldn't it be in the PhysicsMaterial?).